apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableServiceApiClientPartial.cs

```diff
@@ -55,6 +54,42 @@ public partial class BigtableServiceApiSettings
                 totalExpiration: Expiration.FromTimeout(TimeSpan.FromMilliseconds(600000)),
                 retryFilter: IdempotentRetryFilter
             );
+
+        /// <summary>
+        /// <see cref="RetrySettings"/> for calls to <c>BigtableClient.ReadRows</c> when the stream
+        /// of results is prematurely ended.
```

Comment:
"ends prematurely" rather than "is prematurely ended"?

Reply:
Done